bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/utils/AudioSinkUtilsImpl.java

```diff
@@ -66,11 +66,24 @@ public class AudioSinkUtilsImpl implements AudioSinkUtils {
                             .longValue();
                     return startTime + computedDuration;
                 } catch (IOException | UnsupportedAudioFileException e) {
-                    logger.debug("Cannot compute the duration of input stream", e);
+                    logger.debug("Cannot compute the duration of input stream with method java stream sound analysis",
+                            e);
+                    Integer bitRate = audioFormat.getBitRate();
+                    Integer channel = audioFormat.getChannels();
```

Comment:
channel is apparently not used, is it normal ?

Reply:
Good catch. I used this on a previous try but it is not needed.
I will remove it.